tests/pytests/test_psi4_qcschema.py

```diff
@@ -115,14 +115,14 @@ def test_psi4_scfproperty():
         'MAYER_INDICES', 'MO_EXTENTS', 'GRID_FIELD', 'GRID_ESP', 'ESP_AT_NUCLEI', 'MULTIPOLE(5)', 'NO_OCCUPATIONS'
     ]
 
-    jatin = """{"id": null, "schema_name": "qcschema_input", "schema_version": 1, "molecule": {"schema_name": "qcschema_molecule", "schema_version": 2, "validated": true, "symbols": ["C", "H", "H"], "geometry": [0.0, 0.0, -0.12548920214507722, 0.0, -1.6762075450210006, 0.7470892156790888, 0.0, 1.6762075450210006, 0.7470892156790888], "name": "CH2", "molecular_charge": 0.0, "molecular_multiplicity": 3, "masses": [12.0, 1.00782503223, 1.00782503223], "real": [true, true, true], "atom_labels": ["", "", ""], "atomic_numbers": [6, 1, 1], "mass_numbers": [12, 1, 1], "fragments": [[0, 1, 2]], "fragment_charges": [0.0], "fragment_multiplicities": [3], "fix_com": false, "fix_orientation": false, "provenance": {"creator": "QCElemental", "version": "v0.17.0+7.gf55d5ac.dirty", "routine": "qcelemental.molparse.from_string"}}, "driver": "properties", "model": {"method": "scf", "basis": "6-31G**"}, "keywords": {"docc": [2, 0, 0, 1], "e_convergence": 1e-08, "reference": "UHF", "scf_type": "PK", "socc": [1, 0, 1, 0], "function_kwargs": {"properties": ["DIPOLE", "QUADRUPOLE", "MULLIKEN_CHARGES", "LOWDIN_CHARGES", "WIBERG_LOWDIN_INDICES", "MAYER_INDICES", "MAYER_INDICES", "MO_EXTENTS", "GRID_FIELD", "GRID_ESP", "ESP_AT_NUCLEI", "MULTIPOLE(5)", "NO_OCCUPATIONS"]}}, "protocols": {}, "extras": {"wfn_qcvars_only": true}, "provenance": {"creator": "Psi4", "version": "1.4a2.dev1087", "routine": "psi4.driver.p4util.procutil"}}"""
+    jatin = """{"id": null, "schema_name": "qcschema_input", "schema_version": 1, "molecule": {"schema_name": "qcschema_molecule", "schema_version": 2, "validated": true, "symbols": ["C", "H", "H"], "geometry": [0.0, 0.0, -0.12548920214507722, 0.0, -1.6762075450210006, 0.7470892156790888, 0.0, 1.6762075450210006, 0.7470892156790888], "name": "CH2", "molecular_charge": 0.0, "molecular_multiplicity": 3, "masses": [12.0, 1.00782503223, 1.00782503223], "real": [true, true, true], "atom_labels": ["", "", ""], "atomic_numbers": [6, 1, 1], "mass_numbers": [12, 1, 1], "fragments": [[0, 1, 2]], "fragment_charges": [0.0], "fragment_multiplicities": [3], "fix_com": false, "fix_orientation": false, "provenance": {"creator": "QCElemental", "version": "v0.17.0+7.gf55d5ac.dirty", "routine": "qcelemental.molparse.from_string"}}, "driver": "properties", "model": {"method": "scf", "basis": "6-31G**"}, "keywords": {"docc": [2, 0, 0, 1], "e_convergence": 1e-08, "reference": "UHF", "scf_type": "PK", "socc": [1, 0, 1, 0], "function_kwargs": {"properties": ["DIPOLE", "QUADRUPOLE", "MULLIKEN_CHARGES", "LOWDIN_CHARGES", "WIBERG_LOWDIN_INDICES", "MAYER_INDICES", "MAYER_INDICES", "MO_EXTENTS", "ESP_AT_NUCLEI", "MULTIPOLE(5)", "NO_OCCUPATIONS"]}}, "protocols": {}, "extras": {"wfn_qcvars_only": true}, "provenance": {"creator": "Psi4", "version": "1.4a2.dev1087", "routine": "psi4.driver.p4util.procutil"}}"""
```

Comment:
Uh, why don't we just save the json into a separate file and load that, instead of parsing this long string-as-json?

Reply:
not gonna lie, I have no idea. I'm happy to do it if others think it's fine!

Reply:
I think the precedent of using external files for comparison in (py)tests is there - e.g. the `test_fchk_writer.py` looks into its own folder for fchk files. So I'd vote for doing it that way

Reply:
There is a trick where you make a folder with the same name as the test and then use the `datadir` fixture at the top of `test_fchk_writer.py`.